pkg/kubectl/cmd/set/set_image.go

```diff
@@ -225,6 +225,9 @@ func (o *ImageOptions) Run() error {
 			return nil
 		})
 		if transformed && err == nil {
+			if info.VersionedObject != nil {
```

Comment:
this doesn't look right (and we do identical things in other places... why would info.Object not be correct?)

Reply:
Any idea why encoder tries to find apps/v1beta1 ReplicaSet/DaemonSet when there's only extensions/v1beta1 and apps/v1beta2 of those resources?